### PR TITLE
Update Health Privacy Section

### PR DESCRIPTION
**Pull request by mmccool**
- Add (small modification of) agreed-upon sentence for health privacy section saying that relevant privacy standards should be followed
- Add citations of HIPPA, GDPR, and PIPEDA
- Add local references for each of HIPAA, GDPR, and PIPEDA (not in SpecRef database)

Resolves https://github.com/w3c/wot-usecases/issues/50

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-usecases/pull/175.html" title="Last updated on Jan 25, 2022, 3:19 PM UTC (0c3ed48)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-usecases/175/e58da67...0c3ed48.html" title="Last updated on Jan 25, 2022, 3:19 PM UTC (0c3ed48)">Diff</a>

### CODE REVIEW

**Comment by mmccool** [comment]
It would be helpful to add Taki Kamiya (@takuki) as a reviewer (he wrote the original use case) but I could not find him in the github interface; he may have to comment on this issue first.

File: index.html [hunk 1]
```diff
@@ -90,9 +90,24 @@
           date: "..",
           publisher: "..."
         },
-        "NMEA": {
-          title: "National Marine Electronics Association",
-          href: "https://www.nmea.org"
+        "HIPAA": {
```
Comment (mlagally) [inline index.html:93]: Why did you remove NMEA?

**Review by mlagally — CHANGES_REQUESTED** [review]
Please reinstate NMEA

**Review by mlagally — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (mmccool) [reply index.html:93]: It actually was replaced by an updated reference in my other PR.  However, the name was changed to refer to the actual specification rather than just the org (the only NMEA spec in wide use in IoT is that used to get data from GPS devices), so I'll search for other usages and repair them.

**Comment by mmccool** [comment]
Conflicts resolved (minor overlap in local reference syntax).  I confirmed that all NMEA references were updated to NMEA-0183 (the actual standard meant among the many NMEA publishes).  Merging...
